src/qt/transactionfilterproxy.cpp

```diff
@@ -66,9 +66,9 @@ void TransactionFilterProxy::setDateRange(const QDateTime &from, const QDateTime
     invalidateFilter();
 }
 
-void TransactionFilterProxy::setAddressPrefix(const QString &_addrPrefix)
+void TransactionFilterProxy::setSearchString(const QString &_searchString)
 {
-    this->addrPrefix = _addrPrefix;
+    this->searchString = _searchString;
```

Comment:
Nit:
```cpp
if (this->m_search_string == search_string) return;
```

Reply:
I think that would actually make performance worse...?

Reply:
As it is now yes because `QLineEdit` doesn't emit unnecessary signals. But from this method perspective IMO this makes sense, the cost of the condition is way cheaper than invalidating the filter.

Reply:
BTW, we could lower case the stored `m_search_string` to make the filter faster.

Reply:
Ah, good points.

Reply:
On second thought, we should never get to this point if the string hasn't changed, and lowercasing the search string doesn't make the filter faster because label/address aren't lowercased.

Reply:
Imagine if pressing the ESC key erases the search string, pressing it the 2nd time would do nothing if the check is there.

So why `.contains(..., Qt::CaseInsensitive)`?